client/src/AppContext.js

```diff
@@ -22,6 +22,16 @@ export class AppContextProvider extends Component {
       { name: 'wootie' }
     ]
   };
+  componentDidMount() {
+    const token = localStorage.getItem('token');
+
+    // TODO verify token OR get_user pick a strategy
+    if (token) {
+      // get use
```

Comment:
The `get_user` endpoint is done so you can go ahead and add a call to the backend. You'll just need to make a GET request to `/api/user/get_user` with the token in the header and it will send back the user data that the token is associated with.